app/drivers/zephyr/dts/bindings/kscan/zmk,kscan-gpio-matrix.yaml

```diff
@@ -15,8 +15,22 @@ properties:
     type: phandle-array
     required: true
   debounce-period:
+    type: int
+    required: false
+    deprecated: true
+    description: Deprecated. Use debounce-down-ms and debounce-up-ms instead.
+  debounce-down-ms:
+    type: int
+    default: 5
+    description: Debounce time for key press in milliseconds. Must be <= 127. Use 0 for eager debouncing.
+  debounce-up-ms:
     type: int
     default: 5
+    description: Debounce time for key release in milliseconds. Must be <= 127.
+  poll-period-ms:
+    type: int
+    default: 10
+    description: Time between reads in milliseconds when polling is enabled
```

Comment:
Can we make Kconfig options to override these? We definitely don't _need_ Kconfig options to override these, especially since it would globally override all kscans, but they would also be more accessible for people working with zmk-config repos where they may not know how to overlay the kscan definition. Alternatively we could write up documentation, but I generally see personally that ZMK users are much more comfortable updating their .conf files than updating DT. I'd like to hear others' thoughts though!

Reply:
+1. It's not hard to override the device tree stuff once you know how to do it but it's definitely not well documented right now. Plus I'm generally in favor of putting config all in one place whenever possible.

Reply:
After attempting to write documentation that explained how to find the correct node to modify, I'm on board with this. I couldn't find a simple way to create an optional `int` setting in Kconfig though, so I'm not sure if what I came up with is the best way to do this.